src/ui/js/Buttons.js

```diff
@@ -55,6 +55,8 @@ Buttons.showButton = function() {
     break;
   }
 
+  $('title').html(buttonName + '&mdash; Button Men Online');
```

Comment:
Here's another whitespace glitch - you need a space between the button name and the mdash